service/src/main/kotlin/fi/espoo/evaka/invoicing/service/InvoiceGenerator.kt

```diff
@@ -72,6 +73,9 @@ fun Database.Transaction.createAllDraftInvoices(range: DateRange = getPreviousMo
 
     val codebtors = unhandledDecisions.mapValues { (_, decisions) -> getInvoiceCodebtor(this, decisions) }
 
+    val feeThresholds = getFeeThresholds(range.start).find { it.validDuring.contains(range) }
+        ?: error("Missing prices for period ${range.start} - ${range.end}, cannot generate invoices")
```

Comment:
Nothing prevents fee thresholds from changing in the middle of a month, so I would use a deterministic way to pick the fee thresholds.

Reply:
Decided to pick the one that contains `range.start` for now.